src/Trinity.Core/Network/Messaging/TrinityResponse.cs

```diff
@@ -35,7 +35,7 @@ static TrinityResponse()
 
         internal TrinityResponse(int size)
         {
-            Buffer = (byte*)CMemory.malloc((ulong)size);
+            Buffer = (byte*)CMemory.C_malloc((ulong)size);

```

Comment:
@leoxia Memory.malloc is a wrapper to CMemory.C_malloc.
Ideally all calls go to the outer wrapper.